content/projects/linux/_index.md

```diff
@@ -61,23 +65,42 @@ Create an empty file named exercise.txt and move this file to the /tmp direc
 Note: foreach instruction write a script and save it under this task
 
 ## Task 6 Find commands
+1. Write a script to find all files ending with 'pdf' on your computer
+2. Write a second command that takes the result of the previous search and copy  into a folder of your choice. 
+3. Write a command to display files that where modified today.
 
-1. Create a script that searches the home directory for all existing files
-2. change directory to My document, create a file named search_result.text. Save and change directory to Descktop
-3. write a scritpt to retrive all directories,files and subdirectories, write the result to search_result.txt
+### Resources 
 
-## Resources
+1. https://www.geeksforgeeks.org/find-command-in-linux-with-examples/
+2. https://unix.stackexchange.com/questions/70455/how-to-run-find-exec-script
 
-1. https://bit.ly/2VQbkDK
 
 ## Task 7 Installing softwares
 
-## Task 8 Linux documentation how to
+
+1. [Presentation](https://docs.google.com/presentation/d/1nDQW1tgDa8CiV2PDQy8pIVh_UliFqMJjOuPbFHFa8Uc/edit?usp=sharing) 
+
+## Task 8 Linux Processes
+
+1. [Presentation](https://docs.google.com/presentation/d/1vAz1DSB-cqv1fcS9VLKb-4-i32oQYFmT80hOlwBk6hs/edit?usp=sharing)
 
 ## Task 9 File operation
 
+1. [Presentation](https://docs.google.com/presentation/d/1BFpEqqVQtQqVh0yzELQxwZVbmpGKaZnCv_xorJitDiY/edit?usp=sharing)
+
 ## Task 10 Text editor
 
-## Task 11 User environment
+1. Using nano text editor create a file named my_bio.txt
+2. Save the file and close the editor
+3. create a folder named my_files and move my_bio.txt within.
+
+### Resources
+
+1. https://www.lifewire.com/beginners-guide-to-nano-editor-3859002
+
+# Advanced
+
+
+## Task 11 User environment(coming soon)
 
-## Task 12 Bash and basic scripting
```

Comment:
These two tasks still need to be done.  I moved the ticket https://trello.com/c/lCKiKjSa/181-create-better-linux-and-bash-introductory-exercises back into today (in progress)

Reply:
There are a few things missing. If you look on the ticket there are a bunch of different commands that were identified as important. eg `ssh`.

Reply:
We scheduled (yesterday) another workshop on the missing topics during the week of the 24th ( please consult the program outline for more).  

Stuff on the ticket (in red eg `ssh`)  were thrown in randomly, I haven't spotted context where they could fit in. However, I see an opportunity to make use of some of these commands during the knowledge sharing on AWS. If you have any other idea where they can fit in please feel free to share you though.